libs/langchain/pyproject.toml

```diff
@@ -104,7 +104,7 @@ tigrisdb = {version = "^1.0.0b6", optional = true}
 nebula3-python = {version = "^3.4.0", optional = true}
 mwparserfromhell = {version = "^0.6.4", optional = true}
 mwxml = {version = "^0.3.3", optional = true}
-awadb = {version = "^0.3.9", optional = true}
+awadb = {version = ">=0.3.9", optional = true}
```

Comment:
https://python-poetry.org/docs/dependency-specification/#caret-requirements

We don't want to use `^0.3.10, if we don't want users to use 0.3.9.

What is the goal, d we want it bumped just for langchain developers or also end users?

Reply:
We recommand all users to use awadb v0.3.10 instead of v0.3.9. 
The interface and implementation for langchain and awadb are not changed,  v0.3.9 and v0.3.10 remain compatible.